flasher/overlays/installer/usr/local/bin/installer.sh

```diff
@@ -87,6 +87,9 @@ echo "Installing resctl-demo to ${CHOICE}; do not turn your computer off."
 echo "You will be prompted to restart your computer after installation."
 echo ""
 
+#Remove existing MBR
+dd if=/dev/zero of=${CHOICE} bs=512 count=1
```

Comment:
Doesn't writing the image with bmaptool already do this? Is it that the image doesn't contain the first block? If so, can you please add some comment explaining why this is necessary?

Reply:
From last output the final partition layout seems to have 
[protective MBR](https://en.wikipedia.org/wiki/GUID_Partition_Table#Protective_MBR_(LBA_0) )
i.e only one partition of type code EE (GPT) which contains  GPT partitions and table (i.e everything rectl-demo have)

and this MBR is not written by our resctl-demo image. 
In theory bmaptool should be same as dd but I am not fully sure about what's special about this "protective MBR".
We are still investigating and above change is just another try.

Reply:
Some comments in mentioned link is notable which I am trying to make sense of :
> Furthermore, GPT-aware OSes may check the protective MBR and if the enclosed partition type is not of type EEh or if there are multiple partitions defined on the target device, the OS may refuse to manipulate the partition table

Reply:
@htejun I am pretty sure the image we create doesn't overwrite sector 0 (e.g the MBR), the image starts from sector 1.